### PR TITLE
DEVDOCS-4367: docs(checkout) CHECKOUT-6874: added original price for response item object

### PR DESCRIPTION
**Pull request by bc-shawnwang**
## What changed?
- Add `originalPrice` for physical/digital item object in storefront cart/checkout api doc response schema and example
- Add `original_price` for physical/digital item object in storefront cart/checkout api doc response schema and example
- Change the description of listPrice/list_price to make it consistent. 

@bigcommerce/checkout

### CODE REVIEW

**Review by bc-mackxu — APPROVED** [review]

**Review by animesh1987 — APPROVED** [review]

**Comment by snaderiBC** [comment]
Also pinging @bigcommerce/dev-docs for visibility

**Comment by snaderiBC** [comment]
@bc-traciporter

**Comment by snaderiBC** [comment]
Thanks @bc-shawnwang 
Please wait for @bc-traciporter or @bigcommerce/dev-docs team approval before merging.

File: reference/carts.sf.yml [hunk 1]
```diff
@@ -654,7 +654,10 @@ components:
           description: Whether the item is taxable.
         listPrice:
           type: number
-          description: 'Item’s list price, as quoted by the manufacturer/distributor.'
+          description: 'The net item price before discounts and coupons. It is based on the product default price or sale price (if set) configured in BigCommerce Admin.'
+        originalPrice:
+          type: number
+          description: 'Item’s original price, set as product default price in the admin panel'
```
Comment (bc-traciporter) [inline reference/carts.sf.yml:660]:
```suggestion
          description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/carts.sf.yml [hunk 2]
```diff
@@ -817,6 +820,9 @@ components:
         listPrice:
           type: number
           description: The net item price before discounts and coupons. It is based on the product default price or sale price (if set) configured in BigCommerce Admin.
+        originalPrice:
+          type: number
+          description: 'Item’s original price, set as product’s default price in the admin panel'
```
Comment (bc-traciporter) [inline reference/carts.sf.yml:825]:
```suggestion
          description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/carts.v3.yml [hunk 3]
```diff
@@ -1517,9 +1518,12 @@ components:
                   coupon_amount:
                     type: number
                     description: The total value of all coupons applied to this item.
+                  original_price:
+                    type: number
+                    description: 'Item’s original price, set as product’s default price in the admin panel'
                   list_price:
                     type: number
-                    description: 'Item’s list price, as quoted by the manufacturer/distributor. Optional price override.'
+                    description: 'The net item price before discounts and coupons. It is based on the product default price or sale price (if set) configured in BigCommerce Admin.'
```
Comment (bc-traciporter) [inline reference/carts.v3.yml:1526]:
```suggestion
                    description: 'The net item price before discounts and coupons. BigCommerce derives an item's list price from the product default price or, if applicable, the sale price configured in the admin panel.'
```

File: reference/carts.v3.yml [hunk 4]
```diff
@@ -1691,9 +1695,12 @@ components:
...
```
Comment (bc-traciporter) [inline reference/carts.v3.yml:1703]:
```suggestion
                    description: 'The net item price before discounts and coupons. BigCommerce derives an item's list price from the product default price or, if applicable, the sale price configured in the admin panel.'
```

File: reference/carts.v3.yml [hunk 5]
```diff
@@ -2161,9 +2171,12 @@ components:
             coupon_amount:
               type: number
               description: The total value of all coupons applied to this item.
+            original_price:
+              type: number
+              description: 'Item’s original price, set as product’s default price in the admin panel'
             list_price:
               type: number
-              description: 'Item’s list price, as quoted by the manufacturer/distributor. Optional price override.'
+              description: 'The net item price before discounts and coupons. It is based on the product default price or sale price (if set) configured in BigCommerce Admin.'
```
Comment (bc-traciporter) [inline reference/carts.v3.yml:2179]:
```suggestion
              description: 'The net item price before discounts and coupons. BigCommerce derives an item's list price from the product default price or, if applicable, the sale price configured in the admin panel.'
```

File: reference/carts.v3.yml [hunk 6]
```diff
@@ -2161,9 +2171,12 @@ components:
             coupon_amount:
               type: number
               description: The total value of all coupons applied to this item.
+            original_price:
+              type: number
+              description: 'Item’s original price, set as product’s default price in the admin panel'
```
Comment (bc-traciporter) [inline reference/carts.v3.yml:2176]:
```suggestion
              description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/carts.v3.yml [hunk 7]
```diff
@@ -2333,9 +2346,12 @@ components:
         coupon_amount:
           type: number
           description: The total value of all coupons applied to this item.
+        original_price:
+          type: number
+          description: 'Item’s original price, set as product’s default price in the admin panel'
```
Comment (bc-traciporter) [inline reference/carts.v3.yml:2351]:
```suggestion
          description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/checkouts.v3.yml [hunk 8]
```diff
@@ -1068,8 +1071,11 @@ definitions:
                       description: The total value of all coupons applied to this item.
                       type: number
                       format: double
+                    original_price:
+                      type: number
+                      description: 'Item’s original price, set as product’s default price in the admin panel'
```
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:1076]:
```suggestion
                      description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/checkouts.v3.yml [hunk 9]
```diff
@@ -945,8 +945,11 @@ definitions:
...
```
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:950]:
```suggestion
                      description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/checkouts.sf.yml [hunk 10]
```diff
@@ -3916,8 +3946,11 @@ definitions:
                     description: The total value of all coupons applied to this item.
                     type: number
                     format: double
+                  originalPrice:
+                    type: number
+                    description: 'Item’s original price, set as product’s default price in the admin panel'
```
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:3951]:
```suggestion
                    description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/checkouts.sf.yml [hunk 11]
```diff
@@ -3804,8 +3831,11 @@ definitions:
...
```
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:3836]:
```suggestion
                    description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/checkouts.sf.yml [hunk 12]
```diff
@@ -2975,8 +2999,11 @@ definitions:
                             description: The total value of all coupons applied to this item.
                             type: number
                             format: double
+                          originalPrice:
+                            type: number
+                            description: 'Item’s original price, set as product’s default price in the admin panel'
```
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:3004]:
```suggestion
                            description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/checkouts.sf.yml [hunk 13]
```diff
@@ -2866,8 +2887,11 @@ definitions:
...
```
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:2892]:
```suggestion
                            description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

File: reference/carts.v3.yml [hunk 14]
```diff
@@ -1987,9 +1994,12 @@ components:
             coupon_amount:
               type: number
               description: The total value of all coupons applied to this item.
+            original_price:
+              type: number
+              description: 'Item’s original price, set as product’s default price in the admin panel'
```
Comment (bc-traciporter) [inline reference/carts.v3.yml:1999]:
```suggestion
              description: 'Set an item's original price as the same as the product default price in the admin panel.'
```

**Review by bc-traciporter — COMMENTED** [review]
I made some editorial updates to remove passive voice.

File: reference/carts.sf.yml (hunk 2 above)
Comment (bc-shawnwang) [reply reference/carts.sf.yml:825]: Thanks for the feedback, but for this original price, it's actually prodduct's default price which already set in admin panel, and here it's just return that value in response. if begin with `Set ...`, i think they might misunderstand that it tell them to set the price, but they don't need.

**Comment by bc-shawnwang** [comment]
@bc-traciporter changed the description of list price and original price

**Review by bc-traciporter — APPROVED** [review]

**Pull request merged**